### ISSUE TITLE
🐛 Panic `begin <= end (3 <= 2) when slicing `/**/``

### ISSUE THREAD

**Issue by qarmin** [biomejs/biome#5785]
### Environment information

```block
Self compiled 4874007c15e88a1a9c0dd298db5a8f8a8b568d34
Ubuntu 22.04
```

### What happened?

File content(at the bottom should be attached raw, not formatted file - github removes some non-printable characters, so copying from here may not work):
```
/**/class C
```

command
```
timeout -v 150 biome lint TEST___FILE.ts
```

cause this
```
Biome encountered an unexpected error

This is a bug in Biome, not an error in your code, and we would appreciate it if you could report it to https://github.com/biomejs/biome/issues/ along with the following information to help us fixing the issue.

When opening the issue, please provide a minimal reproduction, or identify and share the file/code that triggers it. Without a way to reproduce the error, the error can't be fixed:

Source Location: crates/biome_module_graph/src/jsdoc_comment.rs:35:30
Thread Name: biome::workspace_worker_0
Message: begin <= end (3 <= 2) when slicing `/**/`


##### Automatic Fuzzer note, output status "None", output signal "Some(6)"
```

[compressed.zip](https://github.com/user-attachments/files/19930191/compressed.zip)

### Expected result

No panic

### Code of Conduct

- [x] I agree to follow Biome's Code of Conduct
